solutions/pytest_folder/conftest.py

```diff
@@ -0,0 +1,18 @@
+import pytest
+from selenium import webdriver
+import json
+
+
+@pytest.fixture(scope="function")
+def browser():
+    browser = webdriver.Chrome()
+    browser.implicitly_wait(5)
+    yield browser
+    browser.quit()
+
+
+@pytest.fixture(scope="function")
+def auth_data():
+    with open("auth_keys_stepik.json", "r") as data:
+        auth_data = json.load(data)
+        return auth_data
```

Comment:
Рекомендовал бы использовать [pathlib](https://realpython.com/python-pathlib/) (и [не только я](https://treyhunner.com/2019/01/no-really-pathlib-is-great/)):
```python
    auth_path = Path("auth_keys_stepik.json")
    return json.loads(auth_path.read_text())
```

При этом для универсальной работы и из терминала, и из интерфейса PyCharm можно использовать несколько видоизменённый код:
```suggestion
    auth_path = Path().rglob("auth_keys_stepik.json")
    return json.loads(next(auth_path).read_text())
```

Reply:
> Рекомендовал бы использовать [pathlib](https://realpython.com/python-pathlib/) (и [не только я](https://treyhunner.com/2019/01/no-really-pathlib-is-great/)):
> 
> ```python
>     auth_path = Path("auth_keys_stepik.json")
>     return json.loads(auth_path.read_text())
> ```
> 
> При этом для универсальной работы и из терминала, и из интерфейса PyCharm можно использовать несколько видоизменённый код:

прочитала, пока сложно. Оставила в notion, буду использовать

Reply:
Да, поэтому и написал готовый код. Между прочим, и самому пришлось посидеть
и поразбираться с этим.